src/Hl7.Fhir.Serialization.R4B.Tests/SerializationExcexptionHandlersJsonPoco.cs

```diff
@@ -0,0 +1,875 @@
+﻿using Hl7.Fhir.Model;
+using Hl7.Fhir.Serialization;
+using Hl7.Fhir.Utility;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+
+namespace Hl7.Fhir.Serialization.Tests
+{
+    [TestClass]
+    public class SerializationExceptionHandlersJsonPoco
```

Comment:
This is almost complete duplication of the tests in FhirJsonDeserializationTests.cs - you are just testing whether the operation outcomes are constructed correctly, so these tests are now testing both the parsing (which is already tested) + the mapping to operationoutcome (which was not tested). Feels a bit like a waste of your time.

Reply:
You're basically right, I wanted to recheck all the cases in the old/new parsers to verify differences.
(and still deciding if I want to go back and make the old ones handle all these issues or not - and these would be the tests for them - I did start on the old parsers till I found your new one, and didn't throw the work away)